tools/mkpkg/update_binary-addons

```diff
@@ -134,6 +167,10 @@ for addontxt in $KODI_DIR/cmake/addons/bootstrap/repositories/*-addons.txt ; do
     PROCESSED="${PROCESSED}${ADDON}\n"
   done
 
+  # remove current used cloned git repo
+  # to be uptodate on next run
+  rm -rf "${ADDONS_GIT_DIR}"
+
```

Comment:
Please drop this, we want the addon repo to be kept around

Reply:
Really? Are the files updated if they already do exist? If they get overwritten then they can also be removed, isn't it?

Reply:
Yes, git_clone takes care about that and on subsequent runs only few updates need to be fetched.

Reply:
You know you talk about less than 400kb...
I think it's more "save" to just delete and be sure on next run to be updated.

Reply:
I'd be fine with removal guarded by `KEEP_GIT_DIRS`, like it's done for the other cloned git trees. Please add that as a separate commit though, as it's unrelated to github API.

Keeping git dirs around by default not only saves time on subsequent downloads but also makes it a lot easier to debug stuff if something goes wrong - just cd into the dir and check tags, branches etc.

If you want the dris to be removed then run the update scripts with the `-d` option (which clears `KEEP_GIT_DIRS`)